packages/docker-compose/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.8"
```

Comment:
Shouldn't this docker-compose folder not be a separate package? File structure should be blobstorage/docker-compose/docker-compose.yml?

Reply:
yes

Reply:
@SamiAlHassan please alter it to match the relayer then!